src/models/subgraph-entities/vX/project.ts

```diff
@@ -42,8 +42,13 @@ import { parseTapEventJson, TapEvent, TapEventJson } from '../v1/tap-event'
 type BaseProject = {
```

Comment:
this file is 
- adding trending properties to Project type
- remove TrendingProject type
- bit of cleanup